### ISSUE TITLE
help request: How do you specify order of clusterfilter

### ISSUE THREAD

**Issue by rucciva** [fluent/fluent-operator#693]
### Describe the issue

In my understanding, the clusterfilters will be merged into one fluent bit configuration files. so how do you make sure the order of that merge, e.g. when you need to have a clusterfilter become the last filter in fluentbit config

### How did you install fluent operator?

with YAML applied by argocd & kustomize

### Additional context

-

**Comment by adiforluls** [comment]
Yes, the resources are ordered in a lexicographical order of the resource `metadata.name` currently.

**Comment by smallc2009** [comment]
it should be documented if this is not a bug. it might be very common requirement for rewrite-tag part.

**Comment by joshuabaird** [comment]
This issue is being closed due to inactivity.  Please re-open a new issue if it is still relevant.
